package.json

```diff
@@ -30,6 +30,11 @@
     "turbo": "1.12.5",
     "vitest": "1.4.0"
   },
+  "peerDependencies": {
+    "fuels": ">=0.77.0",
+    "socket.io-client": "4.7.2",
+    "axios": "1.5.1"
+  },
```

Comment:
We can remove this `peerDependencies` block, this is the root `package.json`.

Reply:
i need mantained on dependencies

Reply:
I get your changes on infra and remove this packages of `package.json` 
but i recived a error on build...can check this?

Reply:
@helciofranco i resolved this, removing axios and use fetch